scylla/src/execution/mod.rs

```diff
@@ -1,3 +1,8 @@
 //! This module holds entities that control, customize, trace, and measure statement execution.
 //! This includes:
+//! - execution profiles,
 //! - TODO.
+
+pub mod execution_profile;
+
+pub use execution_profile::ExecutionProfile;
```

Comment:
Commit: "move execution_profile.rs to new execution module"

❓ Why re-export ExecutionProfile here? It belongs in the execution_profile module, right?
If the goal is to ease access to commonly used stuff, then imo we should create a prelude module instead of such duplications.

Reply:
Hmm, I've found this a usual thing to re-export the main entity of a module in its parent module.
This way we avoid to long, verbose paths, such as `b::a::A` - instead we have `b::A`.

Reply:
After the changes (`execution_profile` is in `session` now) I think we should either:
- re-export both `ExecutionProfile` and `ExecutionProfileBuilder` and then make `execution_profile` mod private (it does not define any other public items as far as I can tell).
- re-export neither, keep the mod public.

Rationale: typical use case requires building the profile, and so importing the builder too. I don't see compelling reason to re-export one struct but not the other.

Reply:
> * re-export both `ExecutionProfile` and `ExecutionProfileBuilder` and then make `execution_profile` mod private (it does not define any other public items as far as I can tell).

This is my preferred solution.

Reply:
We also need public access to `ExecutionProfileHandle`, so I'm re-exporting it, too.